eZ/Publish/Core/MVC/Symfony/Security/EventListener/SecurityListener.php

```diff
@@ -254,6 +253,7 @@ private function isMasterRequest( Request $request, $requestType )
      */
     protected function hasAccess( SiteAccess $siteAccess )
     {
-        return $this->securityContext->isGranted( new Attribute( 'user', 'login', array( 'valueObject' => $siteAccess ) ) );
+        // JV: Should this go true AuthorizationCheckerInterface?
+        return $this->repository->canUser( 'user', 'login', $siteAccess );
```

Comment:
-1. Doing so will prevent usage of 3rd party voters.
Furthermore, repository must rely under the hood for separation of concerns :wink: